index.html

```diff
@@ -4,17 +4,82 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
+<body class="bg-image">
+  <header class="bg-green">
+    <div class="container">
+      <div class="row text-white">
+        <img src="images/logo.svg">
+        <h1 class="ml-2">Hungry Panda</h1>
+      </div>
+    </div>
   </header>
-  <main>
+  <main class="m-2">
+    <div class="container form">
+      <div class="row align-items-center mt-5">
+        <h1>Find Restaurants Near You</h1>
+      </div>
+      <div class="row">
+        <div class="col-md-6 col-sm-12">
+          <div class="row align-items-center mt-5">
+            <h2 class="pr-5">Pricing:</h2>
+            <button class="price-button mr-2 btn-sm">$</button>
+            <button class="price-button mr-2 btn-sm">$$</button>
+            <button class="price-button mr-2 btn-sm">$$$</button>
+            <button class="price-button mr-2 btn-sm">$$$$</button>
+          </div>
+        </div>
+        <div class="col-md-6 col-sm-12">
+          <div class="row align-items-center mt-5">
+            <h2 class="pr-5">Distance:</h2>
+            <div>
+              <span>0 mi</span>
+              <input type="range" min="0" max="20" step="2">
+              <span>20 mi</span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row mt-5 pt-lg-5">
+        <div class="col-md-6 col-sm-12">
+          <div class="row align-items-center">
+            <h2 class="pr-5 pr-2">Type of Food:</h2>
+            <input class="food-input" type="text">
+          </div>
+        </div>
+      </div>
+      <div class="row pt-lg-5">
+        <div class="col-md-4 col-sm-12">
+          <div class="row align-items-center justify-content-center mt-4">
+            <button class="btn-lg">Show All Results!</button>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="row align-items-center justify-content-center mt-4">
+            <h2>OR</h2>
+          </div>
+        </div>
+        <div class="col-md-4 col-sm-12">
+          <div class="row align-items-center justify-content-center mt-4">
+            <button class="btn-lg">Pick For Me!</button>
+          </div>
+        </div>
+      </div>
+    </div>
   </main>
   <script src="js/data.js"></script>
   <script src="js/main.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js"
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

Comment:
If you are only using bootstrap for styling, these three `script` elements are unnecessary and are pulling in a **_lot_** of unnecessary code. jQuery is over 10k lines of code, last time I checked.